src/Console/Command/AutoMergeCommand.php

```diff
@@ -120,7 +118,7 @@ private function mergeBranches(Package $package, array $projectConfig)
 
                     $this->io->warning($message);
                     $this->slackClient->attach([
-                        'text' => $message,
+                        'text'  => $message,
```

Comment:
These kind of change are wrong and are not related to this PR. Please revert.

Reply:
I removed these commits, but StyleCI won't let me merge this.

Also tried to manual trigger StyleCI, but it won't help :/

Reply:
This is weird. Array alignment should not be activated.

Are you up to date according to master?

Reply:
Did a fresh rebase for this.

Maybe StyleCI is confused, because we have a [repo config](https://github.com/sonata-project/dev-kit/blob/master/.php_cs) and the [project config](https://github.com/sonata-project/dev-kit/blob/master/project/.php_cs)?

Reply:
The repo config is just a symbolic link to the project config, so it should not be.

Reply:
I did a fresh analyse on master and StyleCI tells nothing: https://styleci.io/repos/55150410